components/TodoList.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useEffect } from "react";
+import { TodoContext } from "../store/store-todo";
+import classes from "../styles/Home.module.css";
+import TodoDetails from "./TodoDetails";
+import { filter } from "../models/todo";
+
+const TodoList = () => {
+  const { todoList, getTodo, Filter } = useContext(TodoContext);
+
+  useEffect(() => {
+    let cleanfunc = true;
+    getTodo();
```

Comment:
you should use variable  `cleanfunc` to check if  `cleanfunc = true`   you should call `getTodo()`